experimental/generation/generator/test/commands/dialog/forms/unittests1.schema

```diff
@@ -0,0 +1,117 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "type": "object",
+    "properties": {
+        "Quantity": {
+            "type": "number",
+            "minimum": 1,
+            "maximum": 10,
+            "$entities": [
+                "number"
+            ]
+        },
+        "Length": {
+            "type": "integer",
+            "minimum": 11,
+            "maximum": 20,
+            "$entities": [
+                "integer"
+            ]
+        },
+        "Complete": {
+            "type": "boolean",
+            "$entities": [
+                "boolean"
+            ]
+        },
+        "NameArray": {
+            "type": "array",
+            "items": {
+                "type": "string"
+            },
+            "$entities": [
+                "personName"
+            ]
```

Comment:
This one is fine since it uses an entity that is not intrinsically connected to string.